src/kvstore/LogEncoder.h

```diff
@@ -19,10 +19,11 @@ enum LogType : char {
     OP_MULTI_REMOVE   = 0x4,
     OP_REMOVE_PREFIX  = 0x5,
     OP_REMOVE_RANGE   = 0x6,
-    OP_ADD_LEARNER    = 0x07,
-    OP_TRANS_LEADER   = 0x08,
-    OP_ADD_PEER       = 0x09,
-    OP_REMOVE_PEER    = 0x10,
+    OP_BATCH_WRITE    = 0x7,
+    OP_ADD_LEARNER    = 0x8,
+    OP_TRANS_LEADER   = 0x9,
+    OP_ADD_PEER       = 0x10,
```

Comment:
0xA? and 0xB?

Reply:
Wow，excellent ！Let me take care of it. thanks.